index.js

```diff
@@ -393,190 +393,203 @@
           }
         };
 
+        /* DEPRECATED */
         this._createScriptTag = function () {
-          if (!accounts || accounts.length < 1) {
-            that._log('warn', 'No account id set to create script tag');
-            return;
-          }
-          if (accounts.length > 1) {
-            that._log('warn', 'Multiple trackers are not supported with ga.js. Using first tracker only');
-            accounts = accounts.slice(0, 1);
-          }
+          that._registerScriptTags();
+          that._registerTrackers();
+        };
+
+        /* DEPRECATED */
+        this._createAnalyticsScriptTag = function () {
+          that._registerScriptTags();
+          that._registerTrackers();
+        };
+
+        this._registerScriptTags = function () {
+          var document = $document[0],
+              protocol = '',
+              scriptSource;
 
           if (created === true) {
-            that._log('warn', 'ga.js or analytics.js script tag already created');
+            that._log('warn', 'Script tags already created');
             return;
           }
 
           if (disableAnalytics === true) {
-            that._log('info', 'Analytics disabled: ' + accounts[0].tracker);
-            $window['ga-disable-' + accounts[0].tracker] = true;
+            accounts.forEach(function (trackerObj) {
+              that._log('info', 'Analytics disabled: ' + trackerObj.tracker);
+              $window['ga-disable-' + trackerObj.tracker] = true;
+            });
           }
 
-          _gaq('_setAccount', accounts[0].tracker);
-          if(domainName) {
-            _gaq('_setDomainName', domainName);
-          }
-          if (enhancedLinkAttribution) {
-            _gaq('_require', 'inpage_linkid', '//www.google-analytics.com/plugins/ga/inpage_linkid.js');
-          }
-          if (trackRoutes && !ignoreFirstPageLoad) {
-            if (removeRegExp) {
-              _gaq('_trackPageview', getUrl());
+          //
+          // Universal Analytics
+          //
+          if (analyticsJS === true) {
+            protocol = hybridMobileSupport === true ? 'https:' : '';
```

Comment:
No. The reason is that the protocol must be HTTPS if hybrid mobile, but otherwise it should be unspecified and use whatever the present protocol is in the browser (that may be HTTPS or HTTP). This could also have been written as:

```
protocol = '';
if (hybridMobileSupport === true) {
  protocol = 'https:';
}
```

That might have made the premise a bit more clear.